packages/api/portfolioDrafts/application/getApplicationStep.ts

```diff
@@ -37,7 +48,7 @@ export async function handler(event: APIGatewayProxyEvent): Promise<APIGatewayPr
       })
     );
     if (!result.Item) {
-      return NO_SUCH_PORTFOLIO_DRAFT;
+      return NO_SUCH_PORTFOLIO_DRAFT_FOUND;
```

Comment:
You should use `NO_SUCH_PORTFOLIO_DRAFT` from `api/utils/errors.ts` instead of redeclaring it here.

Reply:
I added `_FOUND` because there were two `NO_SUCH_PORTFOLIO_DRAFT` being imported into the test file causing a naming collision. I added `_FOUND` to the one found on lines 19-22 of `getApplicationStep.ts`. One has a status code of `404` while the other is `400`. Should we still remove one and accept a 404 status code for both cases?

Reply:
We should keep the `404` one because `404` is the statusCode for `NOT_FOUND`

Reply:
The above is sound thinking, however, this is due to the API spec.

These operations all use http status 404 to indicate when their respective step does not exist/wasn't found.
`GET /portfolioDrafts/{portfolioDraftId}/portfolio`
`GET /portfolioDrafts/{portfolioDraftId}/funding`
`GET /portfolioDrafts/{portfolioDraftId}/application`

For those same operations http status 400 is used to indicate when the requested portfolio draft does not exist/wasn't found.

For example, from `.../application`

>         '404':
>          description: Application Step not found for this Portfolio Draft
>        '400':
>          description: The given Portfolio Draft does not exist

We need the similar messages paired with different status codes for these cases.

Reply:
Thanks for providing that explanation. I added the 400 error to `errors.ts` and updated 
the names with `_404` and `_400` to differentiate.